lib/stream/fake.go

```diff
@@ -29,9 +29,9 @@ type Fake struct {
 
 // Produce a fake stream. Unlike a real stream, this does not call a subprocess
 // --- instead it just reads from a predefined stream (Bytes).
```

Comment:
s/reads from/provides/ -- the thing on the other end of the io.Reader is reading.